sdk/storage/AzureStorageBlob/Source/Operation/ResumableOperationQueue.swift

```diff
@@ -34,13 +34,11 @@ internal protocol ResumableOperationQueueDelegate: ResumableOperationDelegate {
 internal class ResumableOperationQueue {
     // MARK: Properties
 
-    internal lazy var operations: [ResumableOperation] = []
```

Comment:
Removes the "data store" aspect of the queue, making it much closer to just a regular operation queue. I originally thought this was needed to make operations that could pause/resume, but now the necessary info to facilitate that lives in the BlobTransfer and BlockTransfer objects, making this added complexity a source of confusion and bugs.